actions/check-product-changes/poller.js

```diff
@@ -328,7 +353,7 @@ async function poll(params, aioLibs) {
         const allSkus = allSkusResp.data.productSearch.items
           .map(({ productView }) => productView || {})
           .filter(Boolean);
-        
+
         // add new skus to state if any
         for (const sku of allSkus) {
```

Comment:
not from the PR, but I got an error running the runner locally because `sku` here is not a string, but an object containing `{sku, urlKey}`. So we need to do something like:
```
for (const skuObj of allSkus) {
  const { sku } = skuObj;
  ...
}
```

Reply:
Fixed